ArduCopter/compassmot.cpp

```diff
@@ -124,7 +124,7 @@ MAV_RESULT Copter::mavlink_compassmot(mavlink_channel_t chan)
         interference_pct[i] = 0.0f;
     }
 
-    hal.scheduler->expect_delay_ms(5000);
+    EXPECT_DELAY(hal, 5000);
```

Comment:
This should be `EXPECT_DELAY_MS` for unit awareness.

Reply:
done